lib/hets-rabbitmq-wrapper.rb

```diff
@@ -7,18 +7,42 @@ module HetsRabbitMQWrapper
   def self.run
     connection = Bunny.new
     connection.start
+
+    min_parsing_version(connection)
+  end
+
+  def self.instance_version
+    #get version from HetsInstance
+  end
+
+  def self.min_parsing_version(connection)
```

Comment:
This is because of long lines which I had to break. I guess, its okay in this case.